vowpalwabbit/array_parameters_dense.h

```diff
@@ -38,37 +40,38 @@ class dense_iterator
 
 class dense_parameters
 {
- private:
-  weight* _begin;
+private:
+  weight * _begin;
   uint64_t _weight_mask;  // (stride*(1 << num_bits) -1)
   uint32_t _stride_shift;
-  bool _seeded;  // whether the instance is sharing model state with others
+  bool _seeded; // whether the instance is sharing model state with others
 
- public:
+public:
   typedef dense_iterator<weight> iterator;
   typedef dense_iterator<const weight> const_iterator;
   dense_parameters(size_t length, uint32_t stride_shift = 0)
-      : _begin(calloc_mergable_or_throw<weight>(length << stride_shift))
-      , _weight_mask((length << stride_shift) - 1)
-      , _stride_shift(stride_shift)
-      , _seeded(false)
-  {
-  }
+    : _begin(calloc_mergable_or_throw<weight>(length << stride_shift)),
+    _weight_mask((length << stride_shift) - 1),
+    _stride_shift(stride_shift),
+    _seeded(false)
+  { }
 
-  dense_parameters() : _begin(nullptr), _weight_mask(0), _stride_shift(0), _seeded(false) {}
+  dense_parameters()
+    : _begin(nullptr), _weight_mask(0), _stride_shift(0), _seeded(false)
+  {}
 
   bool not_null() { return (_weight_mask > 0 && _begin != nullptr); }
 
-  dense_parameters(const dense_parameters& other) { shallow_copy(other); }
-  dense_parameters(dense_parameters&&) = delete;
+  dense_parameters(const dense_parameters &other) { shallow_copy(other); }
+  dense_parameters(dense_parameters &&) = delete;
```

Comment:
Nit
```suggestion
  dense_parameters(dense_parameters&&) = delete;
```

Reply:
fixed

---
In reply to: [270505493](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1800#discussion_r270505493) [](ancestors = 270505493)